### ISSUE TITLE
Request: make path to doc sets configurable. 

### ISSUE THREAD

**Issue by sagotsky** [dnhgff/vache#1]
The default path couldn't find my docsets. Could it be configurable instead so I don't have to have a symlink just for vache?

Thanks!

**Comment by dnhgff** [comment]
this is [configurable](https://github.com/dnhgff/vache#configuration) by both the `g:vache_default_docset_dir` and `g:vache_filetype_options` variables

**Comment by dnhgff** [comment]
@sagotsky could you please report to me what os you are using, weather you use Dash / Zeal, and whether you have explicitely asked for your docset directory to be in a nonstandard location?

if others have a similar configuration, and vache should be able to guess it, i would prefer for vache to pick up your default directory automagically

**Comment by sagotsky** [comment]
TY for pointing me to the options.

I'm on Debian stable with Zeal.  It's in "~/.local/share/zeal/docsets".  I
didn't do a manually config, so I'm not sure why my zeal was lower cased,
but yours wasn't.  Would a "find ~/.local/share -iname zeal -type d" be
overkill?

**Comment by dnhgff** [comment]
thanks. could you tell me what is the contents of ~/.config/Zeal/Zeal.conf (or perhaps debian has lowercased that too :P)

i am thinking of just parsing the docset directory out of that file, as that would be more robust and allow vache and Zeal to be simultaneously (re)configured (i have opened an issue for this feature #4)

**Comment by sagotsky** [comment]
Yup, I've got that file.  No useful path to a docset in there though.  

I installed this back in march.  Could they have moved file locations in the last few months?  You could always set a minimum version for zeal.

```
[General]
geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x1\0\0\0\0\x5\0\0\0\0\f\0\0\a\x7f\0\0\x3\xff\0\0\x5\x1\0\0\0\r\0\0\a~\0\0\x3\xfe\0\0\0\x1\0\0)
hidingBehavior=systray
hotkey=
httpProxy=
httpProxyPass=
httpProxyPort=0
httpProxyUser=
minFontSize=0
prefixes=@Variant(\0\0\0\x1c\0\0\0\0)
proxyType=0
startupBehavior=window
```

**Comment by dnhgff** [comment]
i have opened a new issue for this. thank you for the repro info :cow:

**Comment by dnhgff** [comment]
@sagotsky i can not mention you in the other issue, but i have a proposed fix on #24 if you would like to try it out for me that would be stellar (and report back on #8, which is the true issue that this branch addresses :P)

**Comment by sagotsky** [comment]
(Sorry for the delay).  No luck.  I set g:vache_default_docset_dir with a full path. I keep getting stuck on the can not read error.  When I copy/paste the setting I can cd into it as I'd expect.
